src/components/Button/index.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import Style from './style.module.scss';
+import style from './style.module.scss';
+import { Icon } from '@scrpoker/components';
 
 interface Props {
-  buttonType: string;
-  iconClassName: string;
+  type: string;
+  icon?: string;
+  className?: string;
 }
 
-const Button: React.FC<Props> = ({ buttonType, iconClassName, children }) => {
+const Button: React.FC<Props> = ({ type, icon, children, className }) => {
+  const classes =
+    className === undefined ? style[type] : style[type] + ' ' + className;
```

Comment:
Should set default value for `className` as empty 
```suggestion
  const classes = `${style[type]} ${className}`
```